privacyidea/api/token.py

```diff
@@ -312,9 +314,31 @@ def init():
     if tokenobject:
         g.audit_object.log({"success": True})
         # The token was created successfully, so we add token specific
-        # init details like the google URL to the response
+        # init details like the Google URL to the response
         init_details = tokenobject.get_init_detail(param, user)
         response_details.update(init_details)
+        # Check if a containerSerial is set and assign the token to the container
+        if "container_serial" in param:
+            container_serial = param.get("container_serial")
+            # check if user is allowed to add tokens to containers
+            try:
+                container_add_token_right = check_container_action(request, action=ACTION.CONTAINER_ADD_TOKEN)
+            except PolicyError:
+                container_add_token_right = False
+                log.error(f"User {user.login} is not allowed to add token {tokenobject.get_serial()} to container "
```

Comment:
i would use an info log entry here (or warning) since this is kind of expected behaviour (the user does not have the right). Warning and Error are usually used to show that something is not working correctly in the server.